pytest.ini

```diff
@@ -1,14 +1,15 @@
 [pytest]
 addopts =
     # Stale/won't fix/missing library
+    --ignore="jupyterbook/content/code_gallery/data_access_notebooks/2016-12-19-exploring_csw.ipynb"
     --ignore="jupyterbook/content/code_gallery/data_access_notebooks/2016-12-22-boston_light_swim.ipynb"
     --ignore="jupyterbook/content/code_gallery/data_access_notebooks/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb"
     --ignore="jupyterbook/content/code_gallery/data_access_notebooks/2017-07-25-HFRadar_currents.ipynb"
     --ignore="jupyterbook/content/code_gallery/data_access_notebooks/2017-12-15-finding_HFRadar_currents.ipynb"
+    --ignore="jupyterbook/content/code_gallery/data_access_notebooks/2019-02-26-hurricane_gis_part01.ipynb"
```

Comment:
The SOS service we use in this notebook was shutdown. We could rewrite it to use another data source but the goal of that notebook was to demo SOS, so I don't believe it is worth it. We do have examples of other sources for these data already in #88